run.go

```diff
@@ -736,6 +731,13 @@ func withInternalLabels(ns, name, hostname, containerStateDir string, networks [
 	if logURI != "" {
 		m[labels.LogURI] = logURI
 	}
+	if len(anonVolumes) > 0 {
+		anonVolumeJSON, err := json.Marshal(anonVolumes)
+		if err != nil {
+			return nil, err
+		}
+		m[labels.AnonymousVolumes] = string(anonVolumeJSON)
```

Comment:
Doesn't it hit 4096 char limit of label when many anonVolumes are specified?

https://github.com/containerd/containerd/blob/261c107ffc4ff681bc73988f64e3f60c32233b37/metadata/containers.go#L292-L296

Reply:
Who needs such number of anon volumes